tests/FamilyHubs.ServiceDirectory.Api.AcceptanceTests/Tests/Steps/ServiceSearchSteps.cs

```diff
@@ -16,64 +16,86 @@ namespace FamilyHubs.ServiceDirectory.Api.AcceptanceTests.Tests.Steps;
 /// </summary>
 public class ServiceSearchSteps
 {
-    readonly ConfigModel config;    
-    private readonly string baseUrl;
-    private ServiceSearchRequest request;
-    private HttpResponseMessage lastResponse;
-    private HttpStatusCode statusCode;
-    private const string serviceSearchPath = "api/metrics/service-search";
-    public ServiceSearchSteps()
-    {
-        config = ConfigAccessor.GetApplicationConfiguration();
-        baseUrl = config.BaseUrl;
-    }
+  readonly ConfigModel config;  
```

Comment:
refactor: should be `private readonly`

`private` because public members shouldn't have access to it
`readonly` because only the constructor should be able to change (set) it